### DISCUSSION TITLE
Has it really won M6 forecasting competition as mentioned in README?

### DISCUSSION THREAD

**Discussion by groverpr** [winedarksea/AutoTS#202] [General]
I was reading announcement from Prof. Spyros Makridakis and didn't see AutoTS or the author of this package in the winners. Can the author please post some proof and more detials of the statement listed in README? 

Post by Spyros Makridakis : https://www.linkedin.com/posts/spyros-makridakis-b2ba5a52_congratulations-to-the-global-winners-of-activity-7028775981133791232-Mlzs?utm_source=share&utm_medium=member_desktop

**Comment by winedarksea** [comment]
Your post definitely has me there, "Galloping Gargoyles" (it's cutoff Galloping Gargo...). I didn't compete as "Team AutoTS" which I thought would sound silly, and also be confusing since other people could also use the package if they wanted, and then it would be confusing which team was being talked about. Regardless you can see my presentation in New York in a couple of weeks and hopefully also a published paper in the IJF journal issue corresponding to the competition, if I finish the paper in time...
